content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -100,10 +78,6 @@ to understand how `X-Forwarded-For` headers and trusted client addresses are det
     $ istioctl install -f topology.yaml
     {{< /text >}}
 
-    {{< idea >}}
```

Comment:
Restarting proxies is still needed IIRC. Are you seeing that the Operator does a rolling upgrade for gateways?

Reply:
Yes i had to restart pod afterwards also

Reply:
Added it back in the example task section